src/table/boot.rs

```diff
@@ -4,7 +4,7 @@ use super::Header;
 use bitflags::bitflags;
 use core::{mem, ptr, result};
 use crate::proto::Protocol;
-use crate::{Event, Guid, Handle, Result, Status};
+use crate::{status, Event, Guid, Handle, Result, Status};
```

Comment:
Not sure if it's best to add a separate line to import the "status" module, or to handle it as a complement to "Status". Prior art from other types which get an associated module of the same name would be useful.

Reply:
Made obsolete by the latest changes.